airflow/serialization/schema.json

```diff
@@ -93,7 +93,7 @@
         "end_date": { "$ref": "#/definitions/datetime" },
         "dagrun_timeout": { "$ref": "#/definitions/timedelta" },
         "doc_md": { "type" : "string"},
-        "_default_view": { "type" : "string"},
+        "default_view": { "type" : "string"},
```

Comment:
This makes this version 2 of the schema sadly, so we now need to think about how we handle versions

Reply:
Do you mean we should update user's exists schema in databases?

Reply:
Yes, it would break the existing Dags in D. To get past it wr can revert the renaming of _default_view

Reply:
I remark here, so we have two ways:
* change schema name to `default_view` and make compatible to existing dags 
* keep in the old name `_default_view`

Reply:
_keep in the old name _default_view_ for now

Reply:
> **keep in the old name _default_view** for now

Ok, I get it.